docker/docker-compose-resetdb.yml

```diff
@@ -29,6 +29,7 @@ services:
             - "${PWD}/dags:/usr/local/airflow/dags"
             - "${PWD}/plugins:/usr/local/airflow/plugins"
             - "${PWD}/requirements:/usr/local/airflow/requirements"
+            - "${PWD}/startup_script:/usr/local/airflow/startup"
```

Comment:
I don't think this is needed. Same as the change in docker-compose-sequential.yml

Reply:
Yeah, I guess it is not needed. But given we added other volumes, I added this too.